src/doctors/doctor.controller.ts

```
@@ -1,41 +1,41 @@
 import { Permissions } from '../common/guards/permission/permission.decorator';
 import { PermissionGuard } from '../common/guards/permission/permission.guard';
 import { ZodValidationPipe } from '../common/pipes/zod-validation.pipe';
 import { PermissionType } from '../permissions/enums/permission.enum';
-import { Staff } from '../staff/entities/staff.entity';
+import { Staff } from '../staffs/entities/staff.entity';
 import { DoctorService } from './doctor.service';
 import {
-  type CreateDoctorReq,
-  CreateDoctorReqZodType,
-  CreateDoctorRes,
+    type CreateDoctorReq,
```

ESLint

[error] Delete `··`

(prettier/prettier)